actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -204,7 +204,7 @@ class Parameters
     #
     # Returns a new array of the values of the parameters.
     delegate :keys, :key?, :has_key?, :values, :has_value?, :value?, :empty?, :include?,
-      :as_json, :to_s, to: :@parameters
+      :as_json, :to_s, :each_key, to: :@parameters
```

Comment:
Is this adding to the public API? If so we need to document like the methods above.

Reply:
Fixed, added docs and a changelog.